src/app/shared/components/table/table.component.html

```diff
@@ -36,7 +39,8 @@
                     key !== 'delete' &&
                     key !== 'details' &&
                     key !== 'payload' &&
-                    key !== 'created'">
+                    key !== 'created' &&
+                    key !== 'checkbox'">
           {{ row[key] }}
         </div>
```

Comment:
Can this be optimized with some sort of hash table - an object where you check the presence of the key (i did not look into the code)?

Reply:
Yes we can consider adding a list of keys in the component itself and optimize this. In any case, new issue :)